Overview.src.html

```diff
@@ -850,90 +852,128 @@ <h2 class="no-num no-toc">
+            and is populated based on an implementation specific discovery
+            mechanism. It is set to most recent result of the algorithm to
+            <span>monitor the list of available presentation displays</span>.
+          </p>
+          <p>
+            While there are live <code>Availability</code> objects, the user
+            agent may <span>monitor the list of available presentation
+            displays</span> continuously, so that pages can use the
+            <code>value</code> property of an <code>Availability</code> object
+            to offer presentation only when there are available displays.
+            However, the user agent may not support continuous availability
+            monitoring; for example, because of platform or power consumption
+            restrictions. In this case the <span>Promise</span> returned by
+            <code>getAvailability()</code> is <span data-anolis-spec=
+            "promguide" title="resolve-reject">rejected</span> and the
+            algorithm to <span>monitor the list of available presentation
+            displays</span> will only run as part of the <span title=
+            "startSession">session start</span> algorithm.
+          </p>
+          <p>
+            When there are no live <code>Availability</code> objects (that is,
+            the <span>set of availability objects</span> is empty), user agents
+            should not <span>monitor the list of available presentation
+            displays</span> to satisfy the <a href=
+            "https://github.com/w3c/presentation-api/blob/gh-pages/uc-req.md#req08-power-saving-friendly">
+            power saving non-functional requirement</a>. To further save power,
+            the UA may also keep track of whether the page holding an
+            <code>Availability</code> object is in the foreground. Using this
+            information, implementation specific discovery of <span title=
+            "presentation-display">presentation displays</span> can be resumed
+            or suspended.
+          </p>
+          <p>
+            Some <span>presentation display</span>s may only be able to display
+            a subset of Web content because of functional, security or hardware
+            limitations. Examples are set-top boxes, smart TVs or networked
+            speakers capable of rendering only audio. We say that such a
+            display is a <dfn>compatible presentation display</dfn> for a
+            <dfn>display availability URL</dfn> if the user agent can
+            reasonably guarantee that the presentation of the URL on that
+            display will succeed.
+          </p>
+          <h4>
+            Monitor the list of available presentation displays
+          </h4>
+          <p>
+            If <span>set of availability objects</span> is non-empty, or there
+            is a pending call to <span title="startSession">start a
```

Comment:
Not: s/call/request?
Request to start a session reads better imho than call to start a session. I would use call with a method name, e.g. call to startSession()